pkg/arvo/mar/sole/effect.hoon

```diff
@@ -63,6 +63,10 @@
         $pro
       %+  frond  %pro
       (pairs vis+b+vis.sef tag+s+tag.sef cad+(tape (purge cad.sef)) ~)
+    ::
+        $tab
+      %+  frond  %tab
+      a+(turn p.sef |=([=cord tan=^tank] (pairs match+s+cord info+(tape ~(ram re tan)) ~)))
```

Comment:
Surely this can be made <80 chars wide. (:

Reply:
I am very unclear where I can break this line such that I don’t get a syntax error x_x

Reply:
:-  %a
%+  turn  p.sef
|=(blah blah blah)

Reply:
Pretty much everywhere once you realize that nearly all irregular syntax is just runes in disguise. For example, this is equivalent:

```hoon
:-  %a
%+  turn  p.sef
|=  [=cord =^tank]
%-  pairs
:~  match+s+cord
    info+(tape ~(ram re tank))
==
```